From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Liav A <liavalb@gmail.com>
Date: Sat, 12 Aug 2023 19:14:23 +0300
Subject: [PATCH] Include config.h in math.h

Otherwise the build will fail due to #define(s) not being included.
---
 lib/math.in.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/math.in.h b/lib/math.in.h
index c30fc202c0d31b7df331a9910dae0842511589ba..2b595710530c99ec4f4a6681e4e2b623716028d1 100644
--- a/lib/math.in.h
+++ b/lib/math.in.h
@@ -33,6 +33,8 @@
 # include <fp.h>
 #endif
 
+#include <config.h>
+
 #ifndef _GL_INLINE_HEADER_BEGIN
  #error "Please include config.h first."
 #endif
